### PR TITLE
more logging for spigot testing stability

### PR DESCRIPTION
**Pull request by jensengar**

### CODE REVIEW

**Comment by millerx** [comment]
I approve any PR with "more logging" in the title.

File: test/unit/sessions/video-session-handler.test.ts [hunk 1]
```diff
@@ -362,7 +362,7 @@ describe('startSession', () => {
       }
     } as any;
 
-    jest.spyOn(utils, 'requestApi').mockResolvedValue({});
+    jest.spyOn(utils, 'requestApi').mockResolvedValue({ body: 'sldk' });
```
Comment (millerx) [inline test/unit/sessions/video-session-handler.test.ts]: `'sldk'` ?

File: spigot-tests/tests/acd-screen-share-test.js [hunk 2]
```diff
@@ -105,17 +105,19 @@ describe('ACD Screen Share via webrtc-sdk [acd-screenshare-sdk] [sdk] [stable]',
     await testUtils.timeout(4000);
 
     const peerStreamAdded = new Promise((resolve, reject) => {
-      setTimeout(() => reject(new Error('Timeout waiting for remote stream')), config.validationTimeout);
+      const timer = setTimeout(() => reject(new Error('Timeout waiting for remote stream')), config.validationTimeout);
```
Comment (djhouseknecht) [inline spigot-tests/tests/acd-screen-share-test.js]: I wonder if the spigot `rejectTimeout()` util would work for this use case? We wouldn't need to manage timers if it did work. If it doesn't work, then we could try to change the util to support our use case.

File: src/sessions/screen-share-session-handler.ts [hunk 3]
```diff
@@ -14,7 +14,7 @@ export default class ScreenShareSessionHandler extends BaseSessionHandler {
   }
 
   // TODO: someday we should do media right before the session accept once we get away from media presence
```
Comment (djhouseknecht) [inline src/sessions/screen-share-session-handler.ts]: I know this `TODO` isn't from your commit, but isn't this taken care of in the new version of the streaming-client?

File: test/unit/sessions/video-session-handler.test.ts (hunk 1 above)
Comment (jensengar) [reply test/unit/sessions/video-session-handler.test.ts]: just a garbage string

File: src/sessions/screen-share-session-handler.ts (hunk 3 above)
Comment (jensengar) [reply src/sessions/screen-share-session-handler.ts]: I believe we can change this now, yes. I'll open a ticket for that.

**Pull request merged**